syntax = "proto3";

package weather;

service WeatherService {
  rpc GetWeather (WeatherRequest) returns (WeatherResponse);
}

message WeatherRequest {
  string city_name = 1;
}

message WeatherResponse {
  string city_name = 1;
  float temperature = 2;
  string description = 3;
  int32 humidity = 4;
  float wind_speed = 5;
}